File: Examples.GridDesktop/CSharp/GridDesktop.Examples/WorkingWithWorksheet/RenameWorksheet.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using Aspose.Cells.GridDesktop;

namespace GridDesktop.Examples.WorkingWithWorksheet
{
    public partial class RenameWorksheet : Form
    {
        public RenameWorksheet()
        {
            InitializeComponent();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            try
            {
                // ExStart:1
                // Accesing an active worksheet directly
                Worksheet sheet = gridDesktop1.GetActiveWorksheet();

                // Renaming a worksheet
                sheet.Name = "Renamed Sheet";
                // ExEnd:1
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message);
            }
        }
    }
}

File: Examples.GridDesktop/VisualBasic/GridDesktop.Examples/WorkingWithWorksheet/RenameWorksheet.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports Aspose.Cells.GridDesktop

Namespace WorkingWithWorksheet
	Public Partial Class RenameWorksheet
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button2_Click(sender As Object, e As EventArgs)
			Try
				' ExStart:1
				' Accesing an active worksheet directly
				Dim sheet As Worksheet = gridDesktop1.GetActiveWorksheet()

                ' Renaming a worksheet
                sheet.Name = "Renamed Sheet"
                ' ExEnd:1
			Catch ex As Exception
				MessageBox.Show(ex.Message)
			End Try
		End Sub
	End Class
End Namespace
